plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiCopyOnWriteConnectorTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.hudi;
+
+import org.apache.hudi.common.model.HoodieTableType;
+
+import static org.apache.hudi.common.model.HoodieTableType.COPY_ON_WRITE;
+
+public class TestHudiCopyOnWriteConnectorTest
+        extends BaseHudiConnectorTest
+{
+    @Override
+    protected HoodieTableType getHoodieTableType()
```

Comment:
General curiosity question:
When should `Hoodie` instead of `Hudi` be used :) ?
I was a bit confused also while stumbling on this term while reading the Hudi documentation.

Reply:
Good question! `Hoodie` was later renamed to `Hudi` project. We changed in the documentation. In the code, we did not change the name in the classes/APIs/configs. So, it's ok to use `Hoodie` in code but not in docs (unless of course the doc is about some API or config key that has `Hoodie` in its name).